docs/flow/modin/experimental/engines/omnisci_on_ray/omnisci_engine.rst

```diff
@@ -0,0 +1,134 @@
+:orphan:
```

Comment:
Do you suggest renaming the file to `docs/flow/modin/experimental/backends/omnisci_on_ray/omnisci_backend.rst`? Wouldn't it be misleading due to the fact that the engine part is described here?

Reply:
I suggest this renaming `docs/flow/modin/experimental/engines/omnisci_on_ray/omnisci_backend.rst`. Currently, Ray is an engine and OmniSci is a backend in Modin terminology. Alsom I suggest replacing `OmniSci engine` to `OmniSci backend` wherever it is applicable.

Reply:
Well, Ray usage is a rudiment we want to get rid of for OmniSci. What would be the engine then? All `OmniSci engine` mentions in this file refer to the execution engine of OmniSciDB and are not related to the Modin codebase.

Reply:
Okay, makes sense to keep as it is (i.e. `OmniSci engine`).

We should define whether it is an engine or is a backend as part of #2957.
If we want to get rid of Ray for OmniSci, directories structure should be reworked because it looks confusing. However, there is an open question related to using Ray or OmniSci Distributed Configuration itself to run the latter on a cluster. I think we should start this discussion.

Reply:
https://github.com/modin-project/modin/issues/3191